pkg/cmd/generate.go

```diff
@@ -64,15 +69,18 @@ func RunGenerate(cmd *cobra.Command, args []string) error {
 		return err
 	}
 
-	// print w/ escaped newlines
+	// Add separator to stringify output
+	stringOutput := "'" + strings.Join(output, "\n\n +----------------------------------------------------------+ \n\n")
+
+	// print stringified output w/ escaped newlines
 	if !r.IsWrite {
-		log.Printf("received output from OpenAI: %s", strings.ReplaceAll(output, "\\n", "\n"))
+		log.Printf("received output from OpenAI: %s", strings.ReplaceAll(stringOutput, "\\n", "\n"))
 	}
 
 	// err = r.Filemap.DecodeFromOutput(output)
 	r.Filemap = filemap.NewFilemap()
 	log.Printf("decoding output")
-	err = r.Filemap.DecodeFromOutput(output)
+	err = r.Filemap.DecodeFromOutput(stringOutput)
```

Comment:
`DecodeFromOutput` is meant for decoding a single response and placing it's contents into a filemap; you would want to call this once for each of the $$n$$ responses OpenAI created.

Reply:
We also only use this method for when we are writing to files, so you wouldn't need to use the concatenated output anyway, that's just for writing to STDOUT.

Reply:
Updated